sfr/abtr_xsgen_workflow/core_hom_macro.i

```diff
@@ -0,0 +1,89 @@
+# ==============================================================================
+# ABTR Equivalent-full core calculation
+# Application: Griffin
+# POC: Shikhar Kumar (kumars at anl.gov)
+# If using or referring to this model, please cite as explained in
+# https://mooseframework.inl.gov/virtual_test_bed/citing.html
+# ==============================================================================
+
+# Input files modifications that MUST be done by the user:
```

Comment:
```suggestion
# This input file is automatically generated by the Griffin EquivalentCoreGenerator
# Input files modifications that MUST be done by the user:
```

Reply:
Updated to:

# This input file is automatically generated by the Griffin [MCC3CrossSection] block when generating the 33 group SFR cross section library